tutorials/02-conda_envs.rst

```diff
@@ -0,0 +1,114 @@
+An Astropy User's Guide to Managing Conda Environments and Jupyter Notebook
+===========================================================================
+
+
+    Help! My code library works in Python 2 and I don't have time to
+    make it work in Python 3
+
+Do not fear! This tutorial will teach you how to use conda environments
+to create a *separate* installation of Python 2.7. With conda
+environments, you can switch between Python 2 and 3 without having to
+worry about version conflicts.
+
+Step 1: Set up a Python 2 environment
+-------------------------------------
+
+-  Create a new ``conda`` environment
+
+   ::
+
+       conda create -n python2 python=2.7 anaconda 
+
+**NOTE:** ``anaconda`` at the end is optional...
+
+-  Activate the Python 2 environment and install any additional packages
+   you need to run your favorite code library. Here, we show you how to 
+   install the base version of Astropy. If you want to download
+   Astropy and **all** affiliated packages, you can install ``stsci``
+
+    ::
+
+        source activate python2
+        conda install astropy
+
+-  When you are ready to return to your default environment:
+
+   ::
+
+       source deactivate
+
+**NOTE:** Some newer versions of Anaconda use ``conda activate`` and
+``conda deactivate``. In that case, both ``source`` and ``conda`` will work 
+interchangeably when activating or deactivating your chosen environment.
+
+When you want to see all of your available environments:
+
+::
+
+    conda env list
+
+Step 2: Check that your code runs in the new environment
+--------------------------------------------------------
+
+-  Now you are ready to work in Python 2! Here's a generic example for
+   switching to your Python 2 environment, running your Python 2 script,
+   and exiting the environment.
+
+    ::
+
+        cd ~/my-python2-library
+        source activate python2
+        python my_python2_script.py
+        source deactivate
+
+Step 3: Set up a Jupyter Notebook for the new environment
+---------------------------------------------------------
+
+-  Activate your custom Python 2 environment:
+
+   ::
+
+       source activate python2
```

Comment:
I would add instructions to install both `notebook` and `ipykernel`. I've been bitten several times by activating an environment in which I forgot to install notebook, running `jupyter notebook` and getting a jupyter not in my environment. Or add notebook to the things to grep below.

Reply:
If they install the anaconda distribution, this shouldn't be an issue, right? ;)

But I hear you, I've run into that problem sometimes, too. I'm adding instructions for installing ipykernel just in case. That's the problem I run into more frequently.

Reply:
Oh wait ... those instructions are already there. Marking as resolved.